### ISSUE TITLE
Feature request: Image to image

### ISSUE THREAD

**Issue by mortenjust** [madebyollin/maple-diffusion#3]
Would be great to be able to prompt the model with a prompt along with an image

**Comment by Kitenite** [comment]
Yes, I would second this. I got this to run on an iPad Pro 4th generation (3.1s/step). I'm so thrilled that this is working!
